features/networking/pod.feature

```diff
@@ -525,3 +525,80 @@ Feature: Pod related networking scenarios
       | ls /var/lib/cni/networks/openshift-sdn/ |
     Then the step should succeed
     And the output should not contain "10.132.2.200"
+
+  # @author zzhao@redhat.com
+  # @case_id OCP-41666
+  # @bug_id 1924741
+  @admin
+  @destructive
+  Scenario: Pod stuck in container creating - failed to run CNI IPAM ADD: failed to allocate for range 0: no IP addresses available in range set
+    Given I switch to cluster admin pseudo user
+    When I run the :label admin command with:
+      | resource  | machineconfigpool         |
+      | name      | worker                    |
+      | key_val   | custom-kubelet=large-pods |
+      | overwrite | true                      |
+    Then the step should succeed
+    Given I obtain test data file "networking/custom_kubelet.yaml"
+    When I run the :create client command with:
+      | f | custom_kubelet.yaml |
+    Then the step should succeed
+    And I wait up to 30 seconds for the steps to pass:
+    """
+    When I run the :get client command with:
+      | resource      | machineconfigpool |
+      | resource_name | worker            |
+    Then the output should match:
+      | .*False\\s+True\\s+False |
+    """
+    And I wait up to 1980 seconds for the steps to pass:
```

Comment:
33 minutes?  yikes.  Do this scale with the number of nodes?

Reply:
yes, this will caused all workers(usually 3)  in this cluster reboot until all workers become ready and scheduled.

Reply:
Updating machineconfigpools is time consuming.   Default behavior is 1-by-1 but we can set maxUnvailable for the machineconfigpool resource to speed it up.   Works for upgrade too.